pdns/pdns_recursor.cc

```diff
@@ -2158,7 +2158,10 @@ static void houseKeeping(void *)
 	try {
 	  doSecPoll(&last_secpoll);
 	}
-	catch(...) {}
+	catch(std::exception& e)
+        {
+          L<<Logger::Error<<"Exception while performing security poll: "<<e.what()<<endl;
+        }
```

Comment:
We should probably keep catching other types of exceptions here, because it looks like a `PDNSException` for example would bubble up and terminate the thread.